lib/slack/messages/message.rb

```diff
@@ -2,10 +2,6 @@
 module Slack
   module Messages
     class Message < Hashie::Mash
-      def presence
```

Comment:
This is an actual Slack field. Aren't we breaking an API here? Should have had tests for it though either way.

Reply:
I checked this a few times, and all the specs are indeed passing, and behavior seems correct.